### PR TITLE
AO3-4560 Rename work posting buttons

### PR DESCRIPTION
**Pull request by niconicosette**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4560

## Purpose

rename work posting buttons to be less confusing

`Preview | Post Without Preview | Cancel`
becomes `Preview | Post | Cancel`

`Save Without Posting | Preview | Post Without Preview | Cancel`
becomes `Save As Draft | Preview | Post | Cancel`

## Testing

1) create an all new work
2) confirm that the changes detailed above are reflected in the work posting options

### CODE REVIEW

**Comment by redsummernight** [comment]
Switching over to Needs Coder Action due to merge conflicts.

**Comment by niconicosette** [comment]
all good? 👍

**Comment by niconicosette** [comment]
travis is still failing but... it doesn't look like it's related to the work posting buttons?

**Comment by redsummernight** [comment]
One of the test failures is related, this notice needs to be updated to match the test:

https://github.com/otwcode/otwarchive/blob/854a4926c67c0419b10b1a4fe2165d8ccb8e9d87/app/controllers/works_controller.rb#L364

Also this comment should be updated:

https://github.com/otwcode/otwarchive/blob/854a4926c67c0419b10b1a4fe2165d8ccb8e9d87/app/controllers/works_controller.rb#L407

**Comment by niconicosette** [comment]
👍 ?

**Comment by redsummernight** [comment]
:+1: !

**Comment by sarken** [comment]
This has merge conflicts again, but no rush to update it!

**Comment by niconicosette** [comment]
all good?

File: features/works/work_edit.feature [hunk 1]
```diff
@@ -75,9 +75,9 @@ Feature: Edit Works
     # Test changing pseuds on a work
     When I go to testuser's works page
       And I follow "Edit"
-      And I select "testy" from "work_author_attributes_ids"
-      And I unselect "testuser" from "work_author_attributes_ids"
-      And I press "Post Without Preview"
+      And I select "testy" from "work_author_attributes_ids_"
```
Comment (sarken) [inline features/works/work_edit.feature]: I think the test failure is due to this and line 79 -- they should be `work_author_attributes_ids`.

**Pull request merged**